src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -39,6 +40,29 @@ public static Index parseIndex(String oneBasedIndex) throws ParseException {
         return Index.fromOneBased(Integer.parseInt(trimmedIndex));
     }
 
+    /**
+     * Parses multiple {@code oneBasedIndex} into {@code Index} and returns them in an array.
+     * Leading and trailing whitespaces will be trimmed.
+     * @throws ParseException if the specified indexes are invalid (not non-zero unsigned integer).
+     */
+    public static Index[] parseMultipleIndex(String oneBasedIndexes) throws ParseException {
+        String trimmedIndexes = oneBasedIndexes.trim();
+        String[] trimmedIndexesList = trimmedIndexes.split("\\s+");
+
+        Index[] indexesList = new Index[trimmedIndexesList.length];
+        for (int i = 0; i < trimmedIndexesList.length; i++) {
+            String trimmedIndex = trimmedIndexesList[i];
+            if (!StringUtil.isNonZeroUnsignedInteger(trimmedIndex)) {
+                throw new ParseException(MESSAGE_INVALID_INDEX);
+            }
+            indexesList[i] = Index.fromOneBased(Integer.parseInt(trimmedIndex));
+        }
+
+        // Sort the array from large to small so that applicants will be deleted from the back
```

Comment:
Would it be better to edit the comment so it won't fit just the delete function?